Detections/ASimNetworkSessions/SeveralDenyActionsRegistered.yaml

```diff
@@ -0,0 +1,40 @@
+id: D66DD819-F2E6-43F7-8BC1-70BDE8011B39
+name: Several deny actions registered (AsimWebSession)
+description: |
+  'Identifies attack pattern when attacker tries to move, or scan, from resource to resource on the network and creates an incident when a source has more than 1 registered deny action in Azure Firewall.'
+tags:
+  - Id: f8dad4e9-3f19-4d70-ab7f-8f19ccd43a3e
```

Comment:
First, I think we discussed renaming this tag, "Id" by itself very confusing. "ParentId" and "ParentVersion"?Secondly, why not have also a "ParentReference". I nearly always look for it to review.

Reply:
ok, but than we need to change in all alert rules/schemas. 
Reference, could be done, but I think both is too much. 
Id: out
ParentReference: In
WDYT?